### PR TITLE
Update Developer Guide

### PR DESCRIPTION
**Pull request by Zack-Tay**
* Add some planned enhancements
* Update MSS for some new commands added in V1.3
* Update appendix for manual testing for some commands.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1072,11 +1100,64 @@ _{More to be added}_
...
+
+**MSS**
+1. User specifies the student id of the student, the module code, tutorial class, and the tutorial team name
+in the tutorial class to allocate the student into.
+2. User enters the command and along with the details.
+3. System allocates student into tutorial team and displays a message. 
+   Use case ends.
+
+**Extensions**
+
```
Comment (whelan-low) [inline docs/DeveloperGuide.md:1122]: Do we need to put invalid input format(s) in the extensions (as part of step 1)?

**Review by whelan-low — APPROVED** [review]
LGTM, just have a small query

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Jajared) [reply docs/DeveloperGuide.md:1122]: Yes invalid input handling should be placed in the extension instead of MSS

**Comment by Jajared** [comment]
@Zack-Tay any amendments to this? If not we can merge and close this PR

**Comment by Zack-Tay** [comment]
> @Zack-Tay any amendments to this? If not we can merge and close this PR

@Jajared If the review is done, this can be merged, I will put another PR for the implemented commands that I have made in the DG.

**Pull request merged**